Composer/packages/server/src/models/asset/assetManager.ts

```diff
@@ -281,8 +282,11 @@ export class AssetManager {
     if (packageName) {
       return packageName
         .replace('generator-', '')
+        .replace('@microsoft/', '')
+        .replace('microsoft', '')
         .split('-')
-        .reduce((a, b) => a.charAt(0).toUpperCase() + a.slice(1) + ' ' + b.charAt(0).toUpperCase() + b.slice(1));
+        .reduce((a, b) => a.charAt(0).toUpperCase() + a.slice(1) + ' ' + b.charAt(0).toUpperCase() + b.slice(1))
```

Comment:
this all seems complicated. what is this for?

Reply:
oh i see it is to generate the "pretty" name.

ok maybe we should try to refactor this in the future to use some other mechanism.